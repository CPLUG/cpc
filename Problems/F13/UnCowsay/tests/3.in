 _______________________________________________________
< <|Rain|> #define struct union /* great space saver */ >
 -------------------------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
