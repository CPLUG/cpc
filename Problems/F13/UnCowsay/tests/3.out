<|Rain|> #define struct union /* great space saver */
